openmetadata-ui/src/main/resources/ui/src/utils/EntityUtils.tsx

```diff
@@ -1794,3 +1794,63 @@ export const columnSorter = (
 export const getColumnNameFromEntityLink = (entityLink: string) => {
   return EntityLink.getTableColumnName(entityLink);
 };
+
+export const getEntityNameLabel = (entityName?: string) => {
+  const entityNameLabels = {
+    table: t('label.table'),
+    topic: t('label.topic'),
+    pipeline: t('label.pipeline'),
+    container: t('label.container'),
+    dashboard: t('label.dashboard'),
+    testCase: t('label.test-case'),
+    testSuite: t('label.test-suite'),
+    ingestionPipeline: t('label.ingestion-pipeline'),
+    all: t('label.all'),
+    announcement: t('label.announcement'),
+    chart: t('label.chart'),
```

Comment:
Why we need this map?

Reply:
added this mapping to show localized options of data assets in the alert and automation form.